datajoint_utilities/dj_search/lists.py

```diff
@@ -79,7 +79,7 @@ def drop_schemas(prefix, dry_run=True, ordered=False, force_drop=False):
             for schema_name in schemas_with_prefix:
                 try:
                     dj.schema(schema_name).drop(force=force_drop)
```

Comment:
We need to modify `drop` to make it raise DataJoint errors rather than pymysql errors.

Reply:
I opened an [issue in datajoint-python](https://github.com/datajoint/datajoint-python/issues/1032) to this effect